### DISCUSSION TITLE
Opening files that are included in .gitignore

### DISCUSSION THREAD

**Discussion by cem-al** [helix-editor/helix#5596] [Q&A] [answered]
Is it possible to open files that are included in .gitignore while having hidden set to false? I don't want `node_modules` included in any search, but occasionally I need to update a `.env` file. Other than opening directly from the terminal with `hx .env` what is the best way for opening a file like an `.env`? 

```
[editor.file-picker]
hidden = false
```

**Comment by the-mikedavis** [comment, marked as answer]
You can open hidden files by giving the full path with `:open .env` within the editor. Hidden files are not included in `:open`'s autocompletion but will be opened.
